app/(nms)/inventory/gnbs_modals.tsx

```diff
@@ -0,0 +1,156 @@
+import { Button, Form, Input, Modal } from "@canonical/react-components"
+import { editGnb } from "@/utils/gnbOperations";
+import { GnbItem } from "@/components/types";
+import { useAuth } from "@/utils/auth";
+import { queryKeys } from "@/utils/queryKeys";
+import { useQueryClient } from "@tanstack/react-query"
+import { useFormik } from "formik";
+import { useState } from "react"
+import { is401UnauthorizedError}  from "@/utils/errors";
+
+import * as Yup from "yup";
+import ErrorNotification from "@/components/ErrorNotification";
+
+
+interface GnbFormValues {
+    name: string;
+    tac: number;
```

Comment:
```suggestion
    tac: string;
```
In the Webconsole endpoint, tac is expected as string.